### PR TITLE
Standardize the use of <br> tags #5618

### LINKED ISSUE #5618: Standardize the use of <br> tags
Currently, the following variations are used: `<br>`, `<br/>` and `<br />` (the first is most correct under HTML5).

The inconsistency causes quite some confusion for new developers (as I once was) who are trying to comply with coding standards.

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #5618

### CODE REVIEW

**Comment by YongJieYongJie** [comment]
@chowyb Assigning you again (it's deeply satisfying to see all `<br/>` `<br />` and even `<br >` being turned into `<br>`

**Comment by chowyb** [comment]
@YongJieYongJie did you assign me this for the sake of catharsis :stuck_out_tongue:

File: src/test/resources/pages/instructorFeedbackAllSessionTypes.html [hunk 1]
```diff
@@ -956,7 +956,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
             First Session #1
           </td>
           <td>
-            <span data-original-title="The feedback session has been created, is visible, and has ended.<br ></span>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
+            <span data-original-title="The feedback session has been created, is visible, and has ended.<br>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
```
Comment (chowyb) [inline src/test/resources/pages/instructorFeedbackAllSessionTypes.html]: `</span>` in the previous tooltip was probably an error, this change should be fine.

File: src/test/resources/pages/instructorFeedbackAllSessionTypesWithHelperView.html [hunk 2]
```diff
@@ -916,7 +916,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
             First Session #1
           </td>
           <td>
-            <span data-original-title="The feedback session has been created, is visible, and has ended.<br ></span>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
+            <span data-original-title="The feedback session has been created, is visible, and has ended.<br>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
```
Comment (chowyb) [inline src/test/resources/pages/instructorFeedbackAllSessionTypesWithHelperView.html]: Same as above, should be fine.

File: src/test/resources/pages/instructorFeedbackDeleteSuccessful.html [hunk 3]
```diff
@@ -861,7 +861,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
             Allow Early Viewing Session #
           </td>
           <td>
-            <span data-original-title="The feedback session has been created, is visible, and has ended.<br ></span>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
+            <span data-original-title="The feedback session has been created, is visible, and has ended.<br>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
```
Comment (chowyb) [inline src/test/resources/pages/instructorFeedbackDeleteSuccessful.html]: ... okay I'm probably not going to point this out any more since it's fine.

**Comment by chowyb** [comment]
Alright, looks good to go!

**Comment by wkurniawan07** [comment]
@YongJieYongJie I'll help you merge this one because I need to merge the hardcoded email one asap :p

**Pull request merged**